docs/BigQueryDelete-action.md

```diff
@@ -0,0 +1,39 @@
+# Google Cloud BigQuery Delete
+
+Description
+-----------
+This plugin deletes BigQuery dataset or tables.
```

Comment:
deletes a BigQuery dataset or table.

Reply:
There should be some description of what happens if the dataset or table does not exist. Does it fail? Does it continue on?